include/pinocchio/parsers/python.hpp

```diff
@@ -0,0 +1,12 @@
+//
+// Copyright (c) 2015-2023 CNRS INRIA
+//
+
+#ifndef __pinocchio_parser_python_hpp__
+#define __pinocchio_parser_python_hpp__
+
+#include "pinocchio/macros.hpp"
```

Comment:
I guess if you included this file, this was for using a macro for displaying a warning, right ?

Reply:
Right, I used the warning macro but after talking to @jcarpent he doesn't agree to deprecate this header.

I will remove this useless include.

Reply:
I don't see the point in not deprecating it, but this is not an issue. 
In this case we can remove the include of macros.hpp